CHANGELOG.md

```diff
@@ -39,8 +39,14 @@
     reporting an invalid length of 20 or 28 when the user input cookie-secret was not that length.
   - Now we will only base64 decode a cookie-secret to raw bytes if it is 16, 24, or 32 bytes long. Otherwise, we will convert
     the direct cookie-secret to bytes without silent padding added.
+- [#412](https://github.com/oauth2-proxy/oauth2-proxy/pull/412)/[#559](https://github.com/oauth2-proxy/oauth2-proxy/pull/559) Allow multiple cookie domains to be specified
```

Comment:
I'm not sure if this should be in the breaking changes since this feature hasn't been release yet. Just in the interest in keeping the breaking changes list as high signal to noise. But not strongly opinionated, so up to you..

Reply:
The change to the config file `cookie_domain` to `cookie_domains` constitutes a breaking change doesn't it? (Same for env?)

Reply:
Yes, never mind. You're right, this should be here.